### PR TITLE
HoC 2024 - Add new tutorial strings

### PR DESCRIPTION
**Pull request by kelbyhawn**
Adds strings for Hour of Code 2024 partner tutorials.

## Links
Jira ticket: [ACQ-2521](https://codedotorg.atlassian.net/browse/ACQ-2521)
Gsheet: [here](https://docs.google.com/spreadsheets/d/1oWivmerA5ZCyMELDXjeTHB3VALANolXFp_HiGkLDgTQ/edit?gid=1244164543#gid=1244164543)

### CODE REVIEW

**Comment by kelbyhawn** [comment]
Hey @breville, can you take a look at this PR and let me know if I'm missing anything? 

I didn't add any `…languages_supported` strings since those weren't added last year. Is that related to [this change](https://github.com/code-dot-org/code-dot-org/commit/fd5908e81dfcf8910a97de9a71122c42fe1adb9b)?

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -10861,6 +10861,25 @@
   tutorial_socimpactofgenaiai_string_detail_programming_languages: "Language independent"
   tutorial_generationai_string_detail_programming_languages: "Blocks, Python"
 
+  tutorial_scratch_iventing_name: "Inventing with Scratch and Gitanjali Rao"
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:10864]: should this be tutorial_scratch_inventing_name? (similar question for other related strings)

Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:10864]: Ah yes! * wishes spell check worked on this file *

**Review by bethanyaconnor — APPROVED** [review]

**Comment by kelbyhawn** [comment]
> Hey @breville, can you take a look at this PR and let me know if I'm missing anything?
> 
> I didn't add any `…languages_supported` strings since those weren't added last year. Is that related to [this change](https://github.com/code-dot-org/code-dot-org/commit/fd5908e81dfcf8910a97de9a71122c42fe1adb9b)?

I'm going to merge this now so we can get these over to Jorge, if I need to update anything I can do it in a separate PR!

**Comment by breville** [comment]
> I didn't add any `…languages_supported` strings since those weren't added last year. Is that related to [this change](https://github.com/code-dot-org/code-dot-org/commit/fd5908e81dfcf8910a97de9a71122c42fe1adb9b)?

That change just made sure that we don't look up the string table while doing a preview, so that you see the strings directly from the gsheet that you are previewing instead.  It should have no impact on regular users of the page.

The comma-separated language list is generated based on language codes courtesy of [this](https://github.com/code-dot-org/code-dot-org/pull/43096) change.

**Pull request merged**